code/UserForm.php

```diff
@@ -117,6 +119,13 @@ trait UserForm
         'Submissions' => SubmittedForm::class,
     ];
 
+    /**
+     * @var array
+     */
+    private static $has_one = [
```

Comment:
```suggestion
    private static array $has_one = [
```
New API should be strongly typed instead of relying on PHPDocs.